content/millennium/r4/conformance/structure-definition.md

```diff
@@ -106,6 +106,7 @@ Authorization is not required.
  [`related-person-encounter`]                                | Reference to the Encounter associated to the encounter level RelatedPerson.
  [`relationship-level`]                                      | The resource's relationship to either the patient or encounter level.
  [`replaced-by`]                                             | The resource containing this link must no longer be used. The link points forward to another resource that must be used in lieu of the resource that contains this link.
+ [`replacing`]                                                | A reference to a resource that this resource is replacing.
```

Comment:
This is spaced over once more than everything else.

Reply:
updated [here](https://github.com/cerner/fhir.cerner.com/pull/626/commits/c8e893c718e3d46fdf3d29cf2ab1e5107f7dc9e1#diff-f10dc1eb3735aa1264e82ae17b8a0cd967fc69912568786cd6406be319d3fd17R109)